sparkmagic/sparkmagic/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.8.5'
+__version__ = '0.8.6'
```

Comment:
I think we should bump to 0.9.0.

You will also need to bump autoviz and hdijupyterutils